components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIManagerConfiguration.java

```diff
@@ -572,6 +573,18 @@ private void readChildElements(OMElement serverConfig,
                 setExtensionListenerConfigurations(element);
             } else if (APIConstants.JWT_AUDIENCES.equals(localName)){
                 setRestApiJWTAuthAudiences(element);
+            } else if ("APIKeyConfigs".equals(localName)) {
```

Comment:
move `APIKeyConfigs` string to `APIConstants` and refer.

Reply:
Done.